src/components/portal/partial-styles/portal-container.scss

```diff
@@ -13,3 +13,59 @@
         display: inline-block;
     }
 }
+
+@keyframes fade-in {
+    from {
+        opacity: 0;
+        transform: scale(0.94);
+    }
+    to {
+        opacity: 1;
+        transform: scale(1) translate3d(0, 0, 0);
+    }
+}
+
+@keyframes fade-out {
+    from {
+        opacity: 1;
+        transform: scale(1);
+    }
+    to {
+        opacity: 0;
+        transform: scale(0.94) translate3d(0, 0, 0);
+    }
+}
+
+@mixin transform-origin($position, $origin) {
+    &[data-popper-placement='#{$position}'] {
+        > * {
+            transform-origin: $origin;
+        }
+    }
+}
+
+.limel-portal--container {
+    > * {
+        // The element that the consumer component puts inside the portal
+        animation: fade-in 0.18s cubic-bezier(0.25, 0.8, 0.25, 1) forwards;
```

Comment:
#### Why animating the element inside the container, instead of the container itself?

This is because the container has some inlined-styles such as `transform` that are dynamically set by portal. I am trying to animate the same thing, which interferes with those inlined styles. So the next best thing is to animate the content of the container instead.